app/views/analytics/index.html.erb

```diff
@@ -153,11 +153,45 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
+  <!-- Editors List-->
+    <div class="col-sm-4">
       <div class="chart-wrapper">
         <div class="chart-title background-orange">
+          Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+        </div>
+        <div class="chart-stage">
+         <table class='table leader-list'>
+           <tr>
+            <th></th>
+            <th>Editor</th>
+            <th>Updates</th>
+           </tr>
+
+           <% case_updaters_count(10).each_with_index do |who, i| %>
```

Comment:
So, I'm not a fan of having a column dedicated to numbers - below we say it's the ten most active editors in the past 30 days.  I say let's change the title to state this, get rid of the column number, and we can also get rid of the chart notes below unless we need it for another reason.  That should also simplify this from `each_with_index` to `each`.

Reply:
This is easy to change and I don't have a strong view on the number column. The note on the bottom is aimed at providing a space for some additional explanation while maintaining relatively simple chart titles. @trystant -- do you have a view either way?